### ISSUE TITLE
Error - Color Zones + Srf by EP Result

### ISSUE THREAD

**Issue by arkdanielnielsen** [mostaphaRoudsari/Honeybee#244]
Hi @chriswmackey 
I dont understand way the color component comes with the attached error. It is just a simple box.
Zones: something in line 558 and 666
Srf: line 539 and 637
![honeybee_colorzonessrfbyepresult-01](https://cloud.githubusercontent.com/assets/7299731/5805690/7ac64ac8-a011-11e4-8c6f-a3f4d4059e8b.jpg)

**Comment by arkdanielnielsen** [comment]
Should I post these kind of questions in the GH forum instead? Is that better?

**Comment by mostaphaRoudsari** [comment]
Hi @arkdanielnielsen! Either is good. The errors like this usually is the result of having an older version of Honeybee flying and the fact that the component doesn't do the version check. Have you tried to let the latest version of Honeybee_Honeybee fly and try again?

**Comment by chriswmackey** [comment]
@arkdanielnielsen and @mostaphaRoudsari ,

Yes, this is my fault for not putting in a Ladybug version check.  If you have an updated version of Ladybug on your canvass, the components should run fine.  Let me know otherwise and, if so, we can hopefully close out this issue.

-Chris

**Comment by arkdanielnielsen** [comment]
Thanks @chriswmackey that was the problem :/ I am wondering if it possible to do something so you dont get this kind of errors? Probably first when the primary components gets more stable?

**Comment by arkdanielnielsen** [comment]
By the way, I though @chriswmackey had fixed the challenge with curved surfaces. It writes it does not support curved surfaces?

**Comment by mostaphaRoudsari** [comment]
@arkdanielnielsen we already have a method to check the versions and avoid this issue. I think @chriswmackey will add it to the component soon, and I  think you're right with the issue. Chris has already addressed that.

**Comment by chriswmackey** [comment]
@arkdanielnielsen , yes, there is an easy way to avoid the problem, which is just that I need to use the compatibility check that @mostaphaRoudsari  set up.  I did not know how to use it correctly when I first made the commit of that component but it is now all good.  I also took out the warning about curved surfaces as you are right that the issue is addressed.
Given this, it seems ok to close out the issue.
